mock/cpp_mock_scenarios/src/speed_planning/request_speed_change_relative.cpp

```diff
@@ -67,6 +67,7 @@ class RequestSpeedChangeRelativeScenario : public cpp_mock_scenarios::CppScenari
         34741, 10.0, 0.0, api_.getHdmapUtils()),
       getVehicleParameters());
     api_.setLinearVelocity("front", 3);
+    api_.updateFrame();
```

Comment:
This function is intended to be called on the base type and is not expected to be called within scenarios. Access rights modifications will be implemented in a separate PR, so no changes are needed in this PR.

Reply:
This line is needed in order to pass this particular test. Without it `other_status_` variable is not updated and does not contain other entities. Empty `other_status_` results in a throw when calculating the absolute speed from `speed_change::RelativeTargetSpeed` variable.

Reply:
If so, you should wait for one frame to elapse before making a decision, and should not implement the decision on the spot.

Reply:
i have moved `requestSpeedChange` to `onUpdate`